### DISCUSSION TITLE
Make a speedtest fail for testing

### DISCUSSION THREAD

**Discussion by svenvg93** [alexjustesen/speedtest-tracker#1543] [Q&A] [answered]
As im working on adding notification for when a speedtest fails. Does anyone know a good way to make it fail and being marked as failed. 
When I would block internet on the machine at all the notification won't be sent either hahah.

**Comment by svenvg93** [comment, marked as answer]
Nevermind, seems @alexjustesen  is already working on this https://github.com/alexjustesen/speedtest-tracker/discussions/1470#discussioncomment-9708220

**Reply by alexjustesen** [reply]
@svenvg93 architecting this one out for work over the weekend, I'm going to make this as easy as possible to you want it just as an env variable? Something along the lines of `DEV_FAIL_TESTS`?

**Reply by svenvg93** [reply]
Yess that could work. Its just for testing purposes when doing some dev work on it :)
